src/modules/olMap/ol/geodesic/geodesicGeometry.js

```diff
@@ -0,0 +1,241 @@
+			for (let z = 0; z <= arcInterpolationCount; ++z) {
+				const a = daIncrement * z;
+
+				const r = geodesic.ArcPosition(a, Geodesic.STANDARD | Geodesic.LONG_UNROLL);
+				calculatedCoords.push([r.lon2, r.lat2]);
+			}
+			interpolationCount++;
+			return calculatedCoords;
+		};
+
+		geodesicLines.forEach((geodesicLine) =>
+			calculateGeodesicCoordinates(geodesicLine).forEach((c, index) => (index === 0 ? geodesicBag.add(c, true) : geodesicBag.add(c)))
+		);
+		this.#calculationStatus = interpolationCount === 0 ? GEODESIC_CALCULATION_STATUS.INACTIVE : GEODESIC_CALCULATION_STATUS.ACTIVE;
+		return geodesicBag;
+	}
+
+	#createTicksByDistance(distance, map) {
+		const ticks = [];
+		let residual = null;
+		this.#geodesicLines.forEach((geodesicLine) => {
+			const { geodesic } = geodesicLine;
+			let currentResidual = residual;
+			for (let currentDistance = currentResidual ?? distance; currentDistance <= geodesic.s13; currentDistance += distance) {
+				const r = geodesic.Position(currentDistance, Geodesic.STANDARD | Geodesic.LONG_UNROLL);
+				const tickCoordinate = fromLonLat([r.lon2, r.lat2], 'EPSG:3857');
+				const pixel = map.getPixelFromCoordinate(tickCoordinate);
+				ticks.push([...pixel, r.azi2]);
+				currentResidual = geodesic.s13 - currentDistance;
+			}
+			residual = distance - currentResidual;
+		});
+		return ticks;
+	}
+
+	#createGeodesicLines(coordinates) {
+		return coordinates
+			? coordinates.reduce((geodesicLines, toCoordinate, index) => {
+					const fromCoordinate = index > 0 ? coordinates[index - 1] : null;
+					if (fromCoordinate) {
+						const geodesicLine = {
+							geodesic: Geodesic.WGS84.InverseLine(fromCoordinate[1], fromCoordinate[0], toCoordinate[1], toCoordinate[0]),
+							from: fromCoordinate,
+							to: toCoordinate
+						};
+						geodesicLines.push(geodesicLine);
+					}
+					return geodesicLines;
+				}, [])
+			: [];
+	}
+
+	#calculateAzimuthCircle(coords, rotation, length) {
+		const center = coords[0];
+		const arcLength = 360 / FULL_CIRCLE_POINTS;
+		const circleCoords = new TiledCoordinateBag();
+		for (let i = 0; i <= FULL_CIRCLE_POINTS; i++) {
+			const res = GEODESIC_WGS84.Direct(
+				center[1],
+				center[0],
+				//Adding "rotation" to be sure that the line meets the circle perfectly
+				arcLength * i + rotation,
+				length
+			);
+			circleCoords.add([res.lon2, res.lat2]);
+		}
+		return circleCoords.createTiledGeometry();
+	}
+
+	getTicksByDistance(distance) {
+		return this.#createTicksByDistance(distance, this.#map);
+	}
+
+	getCoordinateAt(fraction) {
+		let fractionLength = this.#length * fraction;
+
+		for (const geodesicLine of this.#geodesicLines) {
+			const { geodesic } = geodesicLine;
+			if (geodesic.s13 < fractionLength) {
+				fractionLength -= geodesic.s13;
+			} else {
+				const r = geodesic.Position(fractionLength, Geodesic.STANDARD | Geodesic.LONG_UNROLL);
+				return fromLonLat([r.lon2, r.lat2], 'EPSG:3857');
```

Comment:
Let's either use the constant value `WEBMERCATOR`  or use 'EPSG:3857'. For me, it's ok when we always use
plain strings (EPSG:3857', EPSG:4326) when transforming geometries .